### DISCUSSION TITLE
Welcome to video-subtitle-extractor Discussions!

### DISCUSSION THREAD

**Discussion by YaoFANGUK** [YaoFANGUK/video-subtitle-extractor#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChillyBboy** [comment]
Hello.

I'm new to this program and it looks very promising. Do I need to do anything special to make the GPU powered program work? When I try the GPU exe it ramps up my CPU usage but my GPU stays low. Do I need to do something to make it work?
